### PR TITLE
Reindent websocket *.any.js files

### PR DESCRIPTION
**Pull request by zcorpan**
Command used:
find . -name "*.any.js" | xargs -I {} js-beautify -r -s 2 -n {}

### CODE REVIEW

**Comment by annevk** [comment]
Generally this looks okay, but I don't like that it removes the newline at the end.

**Comment by zcorpan** [comment]
Fixed. Updated first comment with the command used to have newline at the end (`-n`).

**Review by annevk — APPROVED** [review]

File: websockets/Create-blocked-port.any.js [hunk 1]
```diff
@@ -1,81 +1,81 @@
...
-            })
+async_test(t => {
+  const ws = CreateWebSocketWithBlockedPort(__PORT)
+  ws.onerror = t.unreached_func()
+  ws.onopen = t.step_func_done()
+}, 'Basic check');
+// list of bad ports according to
+// https://fetch.spec.whatwg.org/#port-blocking
+[
+  1, // tcpmux
```
[the remark this answers is not in the document]
Comment (zcorpan) [reply websockets/Create-blocked-port.any.js]: Let's leave this as the tool output it.

**Comment by zcorpan** [comment]
OK the chrome build has failed about 4 times now. @jgraham should we merge this despite the https://github.com/w3c/web-platform-tests/issues/10639 issue?

**Pull request merged**